redis_consumer/consumers/base_consumer.py

```diff
@@ -553,6 +553,35 @@ def predict(self, image, model_name, model_version, untile=True):
 
         return image
 
+    def save_output(self, image, redis_hash, save_name, scale):
```

Comment:
```suggestion
    def save_output(self, image, redis_hash, save_name, scale=1):
```

Let's default `scale` to 1 so it can be a optional.